### ISSUE TITLE
[a11y Support] DBEX-TREX: Timely and Truthful Submission Experience - Progress Bar (Figma)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#92617]
> [!NOTE]
> **QUICK SUMMARY:** 
> - Team has completed midpoint, is preparing for staging and preparing for development with engineers in an upcoming sprint, and is looking for feedback on revisions post-midpoint from CAIA a11ys. 
> - When in staging - needing to re-test to verify progress bar @EvanAtCoforma and @coforma-jamie 
> - Priority level will be needed from @artsymartha68 , during our next sync @NaomiPMC 
> - Please refer to this ticket's task list for a definition of done, as it develops.

## Issue Description

>  _**We have passed the midpoint review and have conducted evaluative research on our proposed designs.**_ 

> _**Based on our findings, we have made revisions and are seeking feedback** on content, **accessibility**, and information architecture._ 

> _While the ask is still broad, we have identified specific areas that would benefit from your expertise in refining some rough content, which we can discuss further during our meeting._ 

### Additional Details

The additional details are c/o the Disability Benefits Team (taken from CAIA Intake Ticket #92471)

<details><summary>Toggle to view initiative details and desired outcomes</summary>

### Initiative Details

_Many Veterans do not have a positive experience when submitting a claim for disability compensation. There are multiple issues that describe errors, confusion, a lack of clarity, missing information, and cognitive load that isn't conducive to the level of quality we aspire to provide for Veterans. This feature focuses on improving the submission experience starting at the point when the Veteran clicks "submit" on a disability benefits application on va.gov and ending when the claim is established in VBMS and all initial doc uploads succeed or the submission process ultimately fails. More in [our feature brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/product/feature-briefs/Submission%20Experience.md)._ 

### Desired Outcomes

_We are redesigning touchpoints with Veterans including screens and emails that happen after a Veteran clicks "submit" on a disability benefits application. We are considering success, backup path, and multiple failure scenarios._

_We are / will be coordinating with other teams on this, notably the Veteran Facing Forms team, My VA / Authenticated Experience, and Benefits Management Tools._

#### Defining Success
- _Ensure the expectations of the claim submission timeline are set truthfully for the Veteran._
- _Ensure the Veteran understands (at a basic level) what is happening with their claim submission._
- _Ensure the language we use is straightforward so that Veterans feel understanding submission status is trivial._
- _Ensure an email confirmation is received in a timely manner and contains information the Veteran deems useful._

</details>

## Timing & Current Progress
> We worked with CAIA earlier in the collaboration cycle (https://github.com/department-of-veterans-affairs/va.gov-team/issues/83554) and are requesting to partner with CAIA again for an additional round of copy review.
> 
> We are preparing to hand off to our engineering teams for development **in an upcoming sprint.**

- Completed Design Intent
- Completed Midpoint Review
- Staging `Next` as of 9/10/2024

## About the Team

<details><summary>Toggle to view team contacts and details</summary>

- Team name: Disability Benefits
- OCTO product owner: @emilytheis Emily Theis
- Product name: 526 Disability Benefits Application
- Product manager: @pacerwow JaredPacer
- Slack channel: [benefits-disability-design](https://dsva.slack.com/archives/C053UDWMH7U)
- Dedicated content writer on your team (if you have one): none
- Dedicated a11y specialist on your team (if you have one): @bacitracin Tracy Tran

</details>

## Supporting Artifacts 

- [Initiative Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/product/feature-briefs/Submission%20Experience.md)
- [Timeline and Truthful Epic](https://app.zenhub.com/workspaces/disability-benefits-experience-team-1-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/82076)
- [Figma designs](https://www.figma.com/design/qyTtKDOTHZPGU59PAfqVhq/Submission-Experience?node-id=5626-129729&t=iMjBr9ubAHdeyj7n-1) with comments for CAIA ⭐ 
- [Figma Prototype ](https://www.figma.com/proto/qyTtKDOTHZPGU59PAfqVhq/Submission-Experience?page-id=5626%3A129729&node-id=5626-130341&node-type=FRAME&viewport=-1166%2C-8928%2C0.5&t=nPBreO4YLEmPjYYl-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=5626%3A130341&show-proto-sidebar=1)
- [Research Findings](products/disability/526ez/research/2024-07-Submission-Status/Research-findings.md)

```[tasklist]
### a11y Tasks
- [x] Collaborate with team and setup a time to meet and discuss revisions - Laura W. (CAIA Content) is going to partner with us on this sync.
- [x] **Progress Bar:** Review Figma designs [with comments for CAIA](https://www.figma.com/design/qyTtKDOTHZPGU59PAfqVhq/Submission-Experience?node-id=5626-129729&t=iMjBr9ubAHdeyj7n-1) - @EvanatCoforma to start the initial review (may divvy with another CAIA a11y @coforma-jamie) **Using VADS Component** (Differs from Other Progress Bars) - to test in staging.
- [x] ~a11y Support: Assist with staging artifact - prep `TBD`~
```
```[tasklist]
### Acceptance Criteria
- [x] ~Determine Acceptance Criteria post meeting~
```
```[tasklist]
## Ticket Updates (CAIA Internal)
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_  still needs confirmed with Martha
- [x] Verify Priorty Level with @artsymartha68 in CAIA a11y/OCTO sync with @NaomiPMC
- [x] _Update date in `Last Checked` field_
- [x] _Label with `Actual` (level of effort it took to complete this ticket)_
- [x] _Update date in_ `Closed Date`
```

```[tasklist]
### Related Tickets
- [ ] #92471 
- [ ] #83554
```

**Comment by coforma-jamie** [comment]
Update from the team:

> We actually decided not to go with the progress bar, due to time constraints. Instead we updated the messaging to be more clear as far as what to expect (length of time/please don't refresh the page). I don't think we need anything from CAIA at the moment.

This ticket is ready for closure review @sara-amanda
